internal/operator/config/dcs.go

```diff
@@ -25,7 +25,9 @@ import (
 	"github.com/crunchydata/postgres-operator/internal/config"
 	"github.com/crunchydata/postgres-operator/internal/kubeapi"
 	"github.com/crunchydata/postgres-operator/internal/util"
+	"github.com/iancoleman/orderedmap"
 	log "github.com/sirupsen/logrus"
+	goyaml "gopkg.in/yaml.v2"
```

Comment:
Can we use the YAML library provided by the Kube sig? This is:

```
"sigs.k8s.io/yaml"
```

You can see references in the code.